### PR TITLE
add dnd-character

### PR DESCRIPTION
**Pull request by BNAndras**

### CODE REVIEW

**Comment by BNAndras** [comment]
@ErikSchierboom, for your review

File: config.json [hunk 1]
```diff
@@ -463,6 +463,14 @@
           "text_formatting"
         ]
       },
+      {
+        "slug": "dnd-character",
+        "name": "D&D Character",
+        "uuid": "6f421414-3668-450d-a7ea-bba6d4e0d75f",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 2
```
Comment (ErikSchierboom) [inline config.json:472]: Maybe bump the difficulty a tiny bit?

**Review by ErikSchierboom — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
@BNAndras Could you fix the merge conflict?

**Comment by BNAndras** [comment]
Merge conflict was fixed.

**Pull request merged**